### PR TITLE
Fix a bug of logging in cni isolator.

### PR DESCRIPTION
**Pull request by haoyixin**
the old errno would always be 0, and we can't get the reason that why execute ifconfig failed.

so, we use the return value of spawn for WSTRINGIFYing error in log.

### CODE REVIEW

File: src/slave/containerizer/mesos/isolators/network/cni/cni.cpp [hunk 1]
```diff
@@ -1994,11 +1995,13 @@ int NetworkCniIsolatorSetup::execute()
...
-    if (os::spawn("ifconfig", {"ifconfig", "lo", "up"}) != 0) {
-      cerr << "Failed to bring up the loopback interface in the new "
-           << "network namespace of pid " << flags.pid.get()
-           << ": " << os::strerror(errno) << endl;
-      return EXIT_FAILURE;
+    int spawn = os::spawn("ifconfig", {"ifconfig", "lo", "up"});
+
+    if (spawn != 0) {
+        cerr << "Failed to bring up the loopback interface in the new "
+            << "network namespace of pid " << flags.pid.get()
```
Comment (haosdent) [inline src/slave/containerizer/mesos/isolators/network/cni/cni.cpp]: Add a extra space, need to align "<<"

File: src/slave/containerizer/mesos/isolators/network/cni/cni.cpp [hunk 2]
```diff
@@ -31,6 +31,7 @@
 #include <stout/strings.hpp>
 
 #include <stout/os/constants.hpp>
+#include "common/status_utils.hpp"
```
Comment (haosdent) [inline src/slave/containerizer/mesos/isolators/network/cni/cni.cpp]: Put it after `#include "common/protobuf_utils.hpp"`

**Comment by haosdent** [comment]
Can use `Fixed a bug in the logging of cni isolator.`

File: src/slave/containerizer/mesos/isolators/network/cni/cni.cpp [hunk 3]
```diff
@@ -1994,11 +1995,13 @@ int NetworkCniIsolatorSetup::execute()
...
-      return EXIT_FAILURE;
+    int spawn = os::spawn("ifconfig", {"ifconfig", "lo", "up"});
+
+    if (spawn != 0) {
+        cerr << "Failed to bring up the loopback interface in the new "
+            << "network namespace of pid " << flags.pid.get()
+            << ": " << (spawn < 0 ? "failed to fork process" : WSTRINGIFY(spawn)) << endl;
+        return EXIT_FAILURE;
     }
   }
```
Comment (haosdent) [inline src/slave/containerizer/mesos/isolators/network/cni/cni.cpp]: `Failed to fork process`

**Comment by haoyixin** [comment]
i will close the pr and make a new one.

**Pull request closed without merging**